docs/sphinx/user_guide/feature/view.rst

```diff
@@ -92,21 +92,59 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+Reshape
+^^^^^^^
+The ``RAJA::Reshape`` method takes a pointer to data and extents in each dimension as arguments and returns a ``RAJA::View`` object.
+The reshape method is specialized on a `meta-layout` template parameter, which specifies the stride ordering for the dimensions.
+
+The first meta-layout is based on a right-most layout which follows standard C indexing (right most index has unit stride).
+For example:
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_right_start
+   :end-before: _Reshape_right_end
+   :language: C++
+In the example above a ``RAJA::View`` is returned with right most indexing having unit stride.
+
+The second supported meta-layout is the ``RAJA::layout_left`` which will follow a Fortran style indexing
+in which the left most index has unit stride and the right most has the longest stride. The configuration
+is demonstrated in the following example.
+
+.. literalinclude:: ../../../../examples/reshape.cpp
+   :start-after: _Reshape_left_start
+   :end-before: _Reshape_left_end
+   :language: C++
+
+Finally, users may specify a custom index ordering by providing a ``std::index_sequence`` type as a
+template argument for the ``RAJA::Reshape`` method. A ``RAJA::View`` enumerates the parathesis accessor
+from left to right starting at zero. The ``std::index_sequence`` will hold an ordering of slowest (left most)
+to fastest (right most) index for the ``RAJA::View``. In the following example, unit stride will be given to
+index 1 while index 2 is assumed to have the unit stride.

```

Comment:
What is this saying? What's the parathesis accessor? Which has the unit stride?

Reply:
I was confused by this also. The middle entry in the `std::index_sequence` is zero in the example, which requires explanation, IMO.